### DISCUSSION TITLE
Option to disable automatic update check

### DISCUSSION THREAD

**Discussion by Plexprofile** [Floorp-Projects/Floorp#595] [Feature Request]
The Floorp browser currently offers the option to either Automatically install updates When Floorp is not running, or Check for updates but let you choose to install them. But It would be better to enable the user to decide whether or not to check for updates at all.
![image](https://github.com/Floorp-Projects/Floorp/assets/139643082/5a7b2a03-637d-4258-80e2-219ec912951f)

**Comment by ioc2e3** [comment]
That ...

The installation version has automatic updates ...

Why can't it update automatically in the Portable version ?

.

**Comment by Olexijl** [comment]
This is simple to fix. That is, if you are in Windows.

1. Create a new file with notepad. 
2. Paste inside the following:

Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Mozilla\Floorp]
"DisableAppUpdate"=dword:00000001

3. Save it as floorp_updates_disabler.reg and doubleclick on it to import into registry.

Fixed. Compatible with Windows 10/11.

**Reply by Palpabols** [reply]
> This is simple to fix. That is, if you are in Windows.
> 
>     1. Create a new file with notepad.
> 
>     2. Paste inside the following:
> 
> 
> Windows Registry Editor Version 5.00
> 
> [HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Mozilla\Floorp] "DisableAppUpdate"=dword:00000001
> 
>     3. Save it as floorp_updates_disabler.reg and doubleclick on it to import into registry.
> 
> 
> Fixed. Compatible with Windows 10/11.

I've been using this browser for almost two years without knowing how to get rid of that update pop-up, and your method worked. Thank you so much!

**Reply by PortianLiu** [reply]
> This is simple to fix. That is, if you are in Windows.这很容易解决。也就是说，如果您使用的是 Windows。
> 
>     1. Create a new file with notepad.使用记事本创建新文件。
> 
>     2. Paste inside the following:粘贴到以下内容中：
> 
> 
> Windows Registry Editor Version 5.00Windows 注册表编辑器版本 5.00
> 
> [HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Mozilla\Floorp] "DisableAppUpdate"=dword:00000001[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Mozilla\Floorp] “DisableAppUpdate”=dword：00000001
> 
>     3. Save it as floorp_updates_disabler.reg and doubleclick on it to import into registry.将其另存为 floorp_updates_disabler.reg 并双击它以导入注册表。
> 
> 
> Fixed. Compatible with Windows 10/11.固定。与 Windows 10/11 兼容。

我想知道，在linux（ubantu）中，是否存在类似的方法

**Comment by PeaceByJesus** [comment]
Thanks. And would  changing the 1 at the end of the dword to a zero  (DisableAppUpdate"=dword:00000000)  restore the update?

**Reply by Olexijl** [reply]
> Thanks. And would changing the 1 at the end of the dword to a zero (DisableAppUpdate"=dword:00000000) restore the update?

Yes, exactly. You can create floorp_updates_enabler.reg to make it more comfortable.

**Reply by PeaceByJesus** [reply]
Thanks. I recommend adding that reg key to "Favorites." BTW, I wonder what key actually saves favorites. For I have made quite a few tweaks and with a new install that would be one of the keys I would like to add to a new reg.
